### DISCUSSION TITLE
Pittsburgh (PBZ) / Error 500 / Unexpected Problem

### DISCUSSION THREAD

**Discussion by daniel-kennedy** [weather-gov/api#654] [Q&A] [answered]
Greetings,

When trying to reach the endpoint for the Pittsburgh office, `/gridpoints/PBZ/210,134/forecast`, I am receiving the following:

{
"correlationId": "2a41d0a",
"title": "Unexpected Problem",
"type": "https://api.weather.gov/problems/UnexpectedProblem",
"status": 500,
"detail": "An unexpected problem has occurred. If this error continues, please contact support at nco.ops@noaa.gov.",
"instance": "https://api.weather.gov/requests/2a41d0a"
}

I sent an email to the address provided in the error response, and did get a response back, but it only said 'Updated distribution list. ' I don't know what that really means, but I apologize if posting here somehow duplicates an existing a ticket.

Thanks,
Dan

**Comment by sullynole** [comment]
That grid location appears to be invalid for PBZ.  What lat/lon location are you attempting to retrieve a forecast for?

**Comment by daniel-kennedy** [comment]
Hi John:

I was attempting for Pittsburgh, but in the end just need Washington, PA; the lat/long wasn't working for me with the latter.

Dan

**Reply by sullynole** [reply, marked as answer]
Using this lat/lon for Washington, PA: 40.175,-80.2506

https://api.weather.gov/points/40.175,-80.2506 --> https://api.weather.gov/gridpoints/PBZ/70,53/forecast

**Comment by daniel-kennedy** [comment]
Thank you so much, John. For the record, how can I determine an area's gridpoints? I must have glossed over the documentation too quickly.

Dan

**Reply by sullynole** [reply]
No problem.  Plugging the lat,lon into the /points endpoint is the method to find the correct gridpoints office/gridX,gridY values.  There is a more thorough example on the Examples tab here: https://www.weather.gov/documentation/services-web-api

Hope this helps.
